l1-contracts/contracts/state-transition/StateTransitionManager.sol

```diff
@@ -473,14 +473,15 @@ contract StateTransitionManager is IStateTransitionManager, ReentrancyGuard, Own
     /// @notice Called by the bridgehub during the failed migration of a chain.
     /// @param _chainId the chainId of the chain
     /// @param _assetInfo the assetInfo of the chain
-    /// @param _prevMsgSender the previous message sender
-    /// @param _data the data of the migration
-    function bridgeClaimFailedBurn(
+    /// @param _depositSender the address of that sent the deposit
+    /// @param _stmData the data of the migration
+    function forwardedBridgeClaimFailedBurn(
         uint256 _chainId,
         bytes32 _assetInfo,
-        address _prevMsgSender,
-        bytes calldata _data
+        address _depositSender,
```

Comment:
in other places of the code in this PR we just use `/* */` in place of unused variables and I think it is the right way, let's do it here also

Reply:
i think this is mitigated by using the struct to pass down the info

Reply:
I am not sure what you mean. The field is just not used (it is a part of the interface and so some data is passed to it, yes, but in this specific function the field is not used). I am not sure why lints do not complain